spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java

```diff
@@ -676,6 +681,44 @@ protected void reset(List<ChannelProxy> channels, List<ChannelProxy> txChannels)
 		this.connection = null;
 	}
 
+	@ManagedAttribute
+	public Properties getCacheProperties() {
+		Properties props = new Properties();
+		props.setProperty("cachMode", this.cacheMode.name());
+		synchronized(connectionMonitor) {
+			props.setProperty("channelCacheSize", Integer.toString(this.channelCacheSize));
+			if (this.cacheMode.equals(CacheMode.CONNECTION)) {
+				props.setProperty("connectionCacheSize", Integer.toString(this.connectionCacheSize));
+				props.setProperty("openConnections", Integer.toString(this.openConnections.size()));
+				props.setProperty("idleConnections", Integer.toString(this.idleConnections.size()));
```

Comment:
Would you mind to explain the reason just don't use `"" + this.connectionCacheSize`?

I intend to believe the comment on SO (http://stackoverflow.com/questions/14126792/why-should-i-use-integer-tostring-instead-of-just-printing-the-integer):
> For string concatination expressions like
> `"The number is " + k;`
> javac builds the actual code using `StringBuilder` as
> `new StringBuilder("The number is ")).append(Integer.toString(k)).toString());`